common/src/main/java/org/openmrs/analytics/OpenmrsUtil.java

```diff
@@ -18,8 +18,8 @@
 import ca.uhn.fhir.rest.client.api.IClientInterceptor;
 import ca.uhn.fhir.rest.client.api.IGenericClient;
 import ca.uhn.fhir.rest.client.interceptor.BasicAuthInterceptor;
+import org.hl7.fhir.dstu3.model.Resource;
```

Comment:
I know that, at least for our use cases in Haiti, we'll need to use R4 resources, so it would be great to support both approaches. I'm hesitant to only allow reading from the OpenMRS R3 endpoint.

Reply:
This is good to know, I did not know that switching to DSTU3 is a non-starter for your case. Created #33 to address this.

Reply:
thanks @pmanko and @bashir2